README.md

```diff
@@ -443,6 +443,6 @@ A: No you cannot.<!-- embARC MLI Library must be built by MetaWare Development T
 A: No you cannot. <!--embARC MLI Library must be built by full version of MetaWare Development Tools. Binaries built with MWDT Lite are not compatible with ARC GNU Tools and full MetaWare Development Tools. Read the MWDT Lite documentation for details.-->
 
 ***Q: I can not build and run example application for my Synopsys board (EMSK, IoTDK, etc), what I shall do?***  
-A: It isn't supported at the moment. Currently we recommend only building for VPX and x86 emulation targets. You can use MLI 1.1 for EM/HS targets. <!--If you build for Synopsys boards refer to documentation [embarc.org](https://embarc.org/projects/development-systems/) as a good starting point. -->
+A: It isn't supported at the moment. Currently we recommend only building for VPX and x86 emulation targets. You can use MLI 1.1 for EM/HS targets. <!--If you build for Synopsys boards refer to documentation [embarc.org](https://foss-for-synopsys-dwc-arc-processors.github.io/platforms.html) as a good starting point. -->
```

Comment:
The displayed text between [ ] doesn't have a relation with the url anymore.
you could change it into:
If you build for Synopsys boards refer to [platform documentation](https://foss-for-synopsys-dwc-arc-processors.github.io/platforms.html) as a good starting point.